<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление/редактирование кофе</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f8f4f0;
    font-family: Arial, sans-serif;
}
QLabel {
    color: #4a2c2a;
    font-weight: bold;
}
QLineEdit, QComboBox, QSpinBox, QDoubleSpinBox, QTextEdit {
    padding: 8px;
    border: 2px solid #d7ccc8;
    border-radius: 4px;
    background-color: white;
    font-size: 11px;
}
QPushButton {
    padding: 10px 20px;
    border: none;
    border-radius: 4px;
    font-weight: bold;
    min-width: 100px;
}
QPushButton#saveButton {
    background-color: #4caf50;
    color: white;
}
QPushButton#saveButton:hover {
    background-color: #45a049;
}
QPushButton#cancelButton {
    background-color: #f44336;
    color: white;
}
QPushButton#cancelButton:hover {
    background-color: #da190b;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название сорта:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>Введите название кофе</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="roastLevelLabel">
       <property name="text">
        <string>Степень обжарки:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="roastLevelCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="beanTypeLabel">
       <property name="text">
        <string>Тип кофе:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="beanTypeCombo"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Цена (руб):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="priceSpin">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>50.000000000000000</double>
       </property>
       <property name="value">
        <double>500.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="volumeLabel">
       <property name="text">
        <string>Объем упаковки (г):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="volumeSpin">
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="tasteDescriptionLabel">
       <property name="text">
        <string>Описание вкуса:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="tasteDescriptionEdit">
       <property name="placeholderText">
        <string>Опишите вкусовые характеристики кофе...</string>
       </property>
       <property name="maximumHeight">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
